pymc_marketing/mmm/delayed_saturated_mmm.py

```diff
@@ -374,10 +453,14 @@ def build_model(
                 var=logistic_saturation(x=channel_adstock, lam=lam),
                 dims=("date", "channel"),
             )
+
+            channel_contributions_var = channel_adstock_saturated * beta_channel
+            if self.time_varying_media_effect:
+                channel_contributions_var *= tv_multiplier_media[:, None]
```

Comment:
It is not a strong opinion, but I think that **if we are using the logic of a base contribution, which is then modified**, it would be great to have this join saved in another `Deterministic` variable. So, users can understand their BASE contribution and the latent multiplier effect independently. We would need to have a small extra piece of code, but what do you think?

e.g:
```python
pm.Deterministic(
var = channel_contributions_var * tv_multiplier_media[:, None]
name='varying_contribution' #or something like it
)
```

Reply:
Yeah this is a nice idea. I think I might actually change the time_varying_prior function so it always swings in the positive range. Then when using it, it always works as a multiplier on base contributions. Will also make joining this logic with logic for hierarchical parameters easy.

I'll make this change, it's nice.

Reply:
I agree :)